Sources/SwiftSyntax/gyb_generated/SyntaxFactory.swift

```diff
@@ -626,15 +626,15 @@ public enum SyntaxFactory {
     let raw = RawSyntax.createAndCalcLength(kind: SyntaxKind.floatLiteralExpr,
       layout: layout, presence: SourcePresence.present)
     let data = SyntaxData.forRoot(raw)
-    return FloatLiteralExprSyntax(data)
+    return FloatLiteralExprSyntax(data)!
   }
 
   public static func makeBlankFloatLiteralExpr() -> FloatLiteralExprSyntax {
     let data = SyntaxData.forRoot(RawSyntax.create(kind: .floatLiteralExpr,
       layout: [
       RawSyntax.missingToken(TokenKind.floatingLiteral("")),
     ], length: .zero, presence: .present))
-    return FloatLiteralExprSyntax(data)
+    return FloatLiteralExprSyntax(data)!
   }
```

Comment:
I believe this always crashes because `.floatingLiteral` token is constructed with an empty string.
IMO this function is completely useless in the first place. Could you avoid auto generating this and `makeBlankIntegerLiteralExpr`?

Reply:
Done.